src/azure-cli/azure/cli/command_modules/keyvault/tests/latest/test_keyvault_commands.py

```diff
@@ -105,6 +117,22 @@ def test_keyvault_private_link_resource(self, resource_group):
                  ])
 
 
+class KeyVaultMHSMPrivateLinkResourceScenarioTest(ScenarioTest):
+    @ResourceGroupPreparer(name_prefix='cli_test_hsm_plr_rg')
+    def test_mhsm_private_link_resource(self, resource_group):
+        self.kwargs.update({
+            'hsm': self.create_random_name('cli-test-hsm-plr-', 24),
+            'loc': 'centraluseuap'
+        })
+        _create_hsm(self)
+        self.cmd('keyvault private-link-resource list --hsm-name {hsm}',
```

Comment:
What is the difference between `az network private-link-resource list`?

Reply:
From `--debug` log I can see `az network private-link-resource list` uses `send_raw_request` (`az rest`) but `az keyvault private-link-resource list` uses Track 2 SDK. This lead to duplication and confusion.

Reply:
Yes it's using the same request. Keyvault send the request with SDK and Network send with raw request.